build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -71,12 +71,6 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
-                    AzurePublisherTool.ListBlobs($"{targetContainer}")
```

Comment:
One thing I forgot about - we drop a version-specific file that was deleted here. These will add up unless we delete them here